packages/backend/src/fee/model.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseModel } from '../shared/baseModel'
+
+export enum FeeType {
+  Sending = 'SENDING',
+  Receiving = 'RECEIVING'
+}
+
+export class Fee extends BaseModel {
+  public static get tableName(): string {
+    return 'fees'
+  }
+
+  public static get relationMappings() {
+    return {
+      asset: {
+        relation: BaseModel.BelongsToOneRelation,
+        modelClass: `${__dirname}/../asset/model`,
+        join: {
+          from: 'fees.assetId',
+          to: 'assets.id'
+        }
+      }
+    }
+  }
+
+  public id!: string
+  public assetId!: string
+  public type!: FeeType
+  public createdAt!: Date
+  public activatedAt?: Date
+  public fixedFee!: bigint
+  public percentageFee!: string
```

Comment:
Does `percentageFee` get typed as a string or number when returning it?

Reply:
It gets returned from postgres as a string. I believe this is because postgres decimals _can_ represent a decimal number much larger than the js `number`, but not in this case because the field only has 1 significant digit and 4 decimal places. The gql type is a Float so it gets parsed to string coming in and and to a number going out.

Reply:
Makes sense. However, should `fixedFee` or `percentageFee` be required? If they are nullable during creation (and are not being set to 0)?

Reply:
I suppose they should not be required if they are nullable, although I think we are moving towards requiring them which I suppose should extend all the way to the database.

Reply:
I'm not sure we are moving towards a future where fee structure is required but rather to a future where it is optional and we could have hooks out to the ASE that sets fees more dynamically. 

That being said, I think for the near future we should make those fields required because we want to remove the `/quotes` integration point.